src/screens/WhotoFollow/elements/ItemUser.js

```diff
@@ -52,10 +47,10 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
     alignItems: 'center',
     justifyContent: 'space-between',
-    height: 76,
+    height: dimen.normalizeDimen(64),
     paddingHorizontal: 20,
     width: '100%',
-    flex: 1,
+    flex: 1
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The height of the container has been changed from a static value to a dynamic one using the `dimen.normalizeDimen` function. This is a good practice as it makes the UI more responsive and adaptable to different screen sizes.
```diff
-    height: 76,
+    height: dimen.normalizeDimen(64),
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->